### DISCUSSION TITLE
Performance

### DISCUSSION THREAD

**Discussion by emrikll** [karieinarsson/qubit-routing#8] [Show and tell]
<img width="534" alt="Main3mil" src="https://user-images.githubusercontent.com/80955078/177012426-a18b3f81-7fb2-4061-82d0-0ddb5b5f9266.png">

Här är bilden för körningen på main

**Comment by karieinarsson** [comment]
Rip det där sög ju😭😭

**Comment by emrikll** [comment]
[model-10-3-3_5.zip](https://github.com/karieinarsson/qubit-routing/files/9034191/model-10-3-3_5.zip)

Här är en zippad mapp för main körningen

**Comment by emrikll** [comment]
<img width="518" alt="explor3mil" src="https://user-images.githubusercontent.com/80955078/177041702-7fd6b1cd-5653-4427-83b0-d067719be62e.png">
Bild från pruning_exploration körningen

**Comment by emrikll** [comment]
[model-10-3-3_7.zip](https://github.com/karieinarsson/qubit-routing/files/9034832/model-10-3-3_7.zip)
Här är zip mappen för den körningen

**Comment by karieinarsson** [comment]
well den ee snabb men suger :(
